mappings/net/minecraft/level/source/LevelSource.mapping

```diff
@@ -1,11 +1,17 @@
 CLASS net/minecraft/class_51 net/minecraft/level/source/LevelSource
 	METHOD method_1802 isChunkLoaded (II)Z
+		ARG 1 chunkX
+		ARG 2 chunkZ
 	METHOD method_1803 decorate (Lnet/minecraft/class_51;II)V
 		ARG 1 levelSource
+		ARG 2 chunkX
+		ARG 3 chunkZ
+	METHOD method_1804 (ZLnet/minecraft/class_62;)Z
+		ARG 2 listener
 	METHOD method_1806 getChunk (II)Lnet/minecraft/class_43;
 		ARG 1 x
 		ARG 2 z
-	METHOD method_1808 toString ()Ljava/lang/String;
-	METHOD method_1807 (II)Lnet/minecraft/class_43;
+	METHOD method_1807 getChunkAt (II)Lnet/minecraft/class_43;
```

Comment:
just `getChunk`

Reply:
it conflicts because there's another method with the same name that does the same exact thing. thanks notch